internal/blockchain/tezos/connector/tezosconnect/docker.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2023 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tezosconnect
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/firefly-cli/internal/docker"
+	"github.com/hyperledger/firefly-cli/pkg/types"
+)
+
+func (t *Tezosconnect) GetServiceDefinitions(s *types.Stack, dependentServices map[string]string) []*docker.ServiceDefinition {
+	dependsOn := make(map[string]map[string]string)
+	for dep, state := range dependentServices {
+		dependsOn[dep] = map[string]string{"condition": state}
+	}
+	serviceDefinitions := make([]*docker.ServiceDefinition, len(s.Members))
+	for i, member := range s.Members {
+		serviceDefinitions[i] = &docker.ServiceDefinition{
+			ServiceName: "tezosconnect_" + member.ID,
+			Service: &docker.Service{
```

Comment:
The thing is that the signatory service works with many different kms and everyone should decide which one they prefer.
That's why there is no one in the FF CLI.
The idea is that user run its own service for tx signing using chosen kms and after that uses FF CLI with the '--connector-config' and provided address of this service.

Configuration example:
```
connector:
  blockchain:
    signatory: http://127.0.0.1:6732
```

Reply:
In the future, there is an idea to make a kind of local signing of transactions, something like FF signer.
But at the moment, there are other priorities.

Reply:
Okay. This is fine for now. I think when we go to create automated E2E tests we will need to revisit this though.

Reply:
Of course. 
In fact, the idea of setting up the Signature as part of the FF stack is not that bad, 
because by default we can use local keys for tx signing (similar to FF signer). 
This is not super secure and for production KMS should be used (or HSM), 
but for local use it is a very good option